GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.Designer.cs

```diff
@@ -143,6 +148,20 @@ private void InitializeComponent()
             this.chkShowRelativeDate.Text = "Show relative date instead of full date";
             this.chkShowRelativeDate.UseVisualStyleBackColor = true;
             // 
+            // chkShowRepoCurrentBranch
+            // 
+            this.chkShowRepoCurrentBranch.AutoSize = true;
+            this.tlpnlGeneral.SetColumnSpan(this.chkShowRepoCurrentBranch, 2);
+            this.chkShowRepoCurrentBranch.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.chkShowRepoCurrentBranch.Location = new System.Drawing.Point(3, 26);
+            this.chkShowRepoCurrentBranch.Name = "chkShowRepoCurrentBranch";
+            this.chkShowRepoCurrentBranch.Size = new System.Drawing.Size(314, 17);
+            this.chkShowRepoCurrentBranch.TabIndex = 5;
+            this.chkShowRepoCurrentBranch.Text = "Show current branch of repositories";
+            this.toolTip.SetToolTip(this.chkShowRepoCurrentBranch, "Shows the branch name of the repositories in the dashboard and the recent reposit" +
+        "ories drop down menu");
```

Comment:
I wonder why this tooltip does not appear in the `English.xlf`.

Reply:
Not a translation string, should be changed. TranslationApp do not find such strings

Reply:
I already warned about that: https://github.com/gitextensions/gitextensions/issues/3745#issuecomment-306492000

Reply:
Fixed